client/src/constants/translations/english/menu-items.js

```diff
@@ -2,7 +2,9 @@ export default {
   HOME: 'Home',
   EDIT_CONTENT: 'Edit content',
   EDIT_DETAILS: 'Edit Details',
-  ORGANISATIONS: 'Organisation',
+  ORGANISATIONS: 'Organisations',
   CUSTOMISE: 'Customise',
+  CHANGES: 'Changes',
   LOG_OUT: 'log out',
+  LOG_IN: 'log in',
```

Comment:
where is this used?

Reply:
because the Menu component reads from translate file